### PR TITLE
[interfaces]: Restart networking service after the file is updated

### PR DESCRIPTION
**Pull request by stcheng**
Signed-off-by: Shuotian Cheng <shuche@microsoft.com>

### CODE REVIEW

File: files/image_config/interfaces/interfaces-config.sh [hunk 1]
```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
 
 sonic-cfggen -m /etc/sonic/minigraph.xml -t /usr/share/sonic/templates/interfaces.j2 >/etc/network/interfaces
-ifdown eth0 && ifup eth0
-ifdown lo && ifup lo
-
+service networking restart
```
Comment (jleveque) [inline files/image_config/interfaces/interfaces-config.sh]: Just curious if we prefer to use `service networking restart` or `systemctl restart networking`.

Comment (stcheng) [inline files/image_config/interfaces/interfaces-config.sh]: i think they are here doing the same thing. is `systemctl` newer than `service` command? I could switch to `systemctl`.

Comment (jleveque) [inline files/image_config/interfaces/interfaces-config.sh]: Yes, they do the same thing. `systemctl` is part of systemd, whereas service has been around awhile and has been made to wrap a few different service managers.

**Comment by jleveque** [comment]
Cavium build failed due to no space left on device (Issue https://github.com/Azure/sonic-buildimage/issues/428). I think this is good to merge.

**Comment by jleveque** [comment]
Just adding a note that this change was implemented in order to make sure Vlans are brought up on boot.

**Pull request merged**